public/mockServiceWorker.js

```diff
@@ -2,22 +2,21 @@
 /* tslint:disable */
```

Comment:
This is a library code that is changed automatically on upgrading the library.

More on this here: https://github.com/mswjs/msw

Reply:
Thanks for leaving this note. It helps :)